automation/src/test/scala/org/broadinstitute/dsp/workbench/hamm/automation/AutomationTestConfig.scala

```diff
@@ -1,10 +1,9 @@
-package org.broadinstitute.dsde.workbench.hamm
-package automation
+package org.broadinstitute.dsp.workbench.hamm.automation
```

Comment:
the previous way of importing will import things in core package, which is usually what we want.